# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/test/browser/cookie_filtering_square.png^headers^
# Commit: 33f21aec0e20
# Full Hash: 33f21aec0e20d39623554816399ee76d13d7e08f
# Author: edguloien <eguloien@mozilla.com>
# Date: 2022-08-23 21:47:27
# Regressor Bug: 1713203
# File Overlap Count: 1
# Description:
#   Bug 1713203 - Added check for host, scheme and origin attributes check before cookie change broadcast to content processes r=dveditz,dragana,nika
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D146346
# ==============================================================================

diff -r fe25d4f35ef8 -r 33f21aec0e20 netwerk/test/browser/cookie_filtering_square.png^headers^
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/netwerk/test/browser/cookie_filtering_square.png^headers^	Tue Aug 23 17:36:24 2022 +0000
@@ -0,0 +1,2 @@
+Cache-Control: no-cache
+Set-Cookie: test-cookie=png